File: Content/assets_src/js/model/_usuarioCargo.js
/*
|--------------------------------------------------------------------------
| Model UsuarioCargo
|--------------------------------------------------------------------------
*/

APP.model.UsuarioCargo = {
    
       model: {
            IdCargo : "",
        },
       
    
        constructor : function (_IdCargo)
        {
            var usuarioCargo = {
                IdCargo : _IdCargo
            };
            
            return usuarioCargo;
        }
    
    };
    

File: Dominio/Entidade/UsuarioCargo.cs
namespace Dominio.Entidade
{
    public class UsuarioCargo
    {
        public int IdUsuarioProcesso { get; set; }
        public int IdUsuario { get; set; }
        public int IdCargo { get; set; }
        public virtual Cargo Cargo { get; set; }
        public virtual Usuario Usuario { get; set; }
    } 
}
